crypto/fipsmodule/sha/asm/keccak1600-armv8.pl

```diff
@@ -458,23 +458,27 @@
 	mov	$out,x1
 	mov	$len,x2
 	mov	$bsz,x3
-.Loop_squeeze:
+	cmp x4, #0
```

Comment:
Suggested (in internal document) to keep the logic as per the OpenSSL commit and to interpret `x4` as padded. This seems to me to still skip the extra Keccak.